src/services/balancer/contracts/contracts/vault.ts

```diff
@@ -30,6 +30,7 @@ export default class Vault {
     result = await this.multiCaller.execute(result);
     this.service.multiCaller.call('totalSupply', poolAddress, 'totalSupply');
     this.service.multiCaller.call('decimals', poolAddress, 'decimals');
+    this.service.multiCaller.call('owner', poolAddress, 'getOwner');
```

Comment:
Is the owner likely to change often? Can we instead get this from the subgraph? Reason being, ideally we only fetch the pool attributes that need to be dynamic with onchain calls.

Reply:
It was in the subgraph after all. (And is immutable for core pools.) So now I get it from there rather than on-chain.